### PR TITLE
20210610 add include for current guidance on localized feedback

### PR DESCRIPTION
**Pull request by WilliamDAssafMSFT**

### CODE REVIEW

File: Contribute/includes/note-docsitelocfeedback.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> The email template available at [aka.ms/DocSiteLocFeedback](https://aka.ms/DocSiteLocFeedback) handles localization-related feedback. We do not offer public Github repositories for localized content feedback with the exception of azure-docs, which is open for community contribution for 6 additional languages (Simplified Chinese, French, German, Japanese, Korean, Spanish). For all other repositories and localizations, use the [email template](https://aka.ms/DocSiteLocFeedback) — which receives your input in any language — to report translation issues or to provide technical feedback.
```
Comment (rothja) [inline Contribute/includes/note-docsitelocfeedback.md:2]: @WilliamDAssafMSFT William, I would suggest starting this include not out more directly with the problem and then the solution of the template. Something like this:

```suggestion
> Most localized documentation does not offer the ability to edit or provide feedback through GitHub. To provide feedback on localized content, use the email template available at [aka.ms/DocSiteLocFeedback](https://aka.ms/DocSiteLocFeedback). There is one exception for Azure documentation. There are public localized GitHub repositories for azure-docs that are open for community contribution for 6 languages: Simplified Chinese, French, German, Japanese, Korean, and Spanish. For all other languages and content sets, use the [email template](https://aka.ms/DocSiteLocFeedback)—which receives your input in any language—to report translation issues or to provide technical feedback. 
```

**Review by rothja — COMMENTED** [review]
I have a suggestion for improving the include file. Thanks!

**Comment by rothja** [comment]
@Jim-Parker Jim, adding you to this PR for helping our international customers understand their editing and feedback options, which have changed recently. Thanks! #assign:Jim-Parker

**Comment by Jim-Parker** [comment]
@WilliamDAssafMSFT @rothja look good to me. If this is ready to merge please change the draft status and sign off.

File: Contribute/includes/note-docsitelocfeedback.md (hunk 1 above)
Comment (WilliamDAssafMSFT) [reply Contribute/includes/note-docsitelocfeedback.md]: I like it

**Comment by WilliamDAssafMSFT** [comment]
CC @jim-parker @rothja 
#sign-off

**Review by rothja — APPROVED** [review]

**Comment by WilliamDAssafMSFT** [comment]
#sign-off

**Comment by WilliamDAssafMSFT** [comment]
@Jim-Parker unfamiliar with this repo's publishing process obviously heh, I don't have a green merge button, can you?

**Review by Jim-Parker — APPROVED** [review]

**Pull request merged**